### DISCUSSION TITLE
Calendar 5 days - how to show full week

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#15010] [How to] [answered]
Somehow calendar hiding Sat-Sun. Dont see settings to show full week? Missed something?
T.

**Comment by vovpff** [comment, marked as answer]
Check calendar cofnfig files

**Reply by s2b-git** [reply]
I did: public static $HIDDEN_DAYS_IN_CALENDAR_VIEW = [8, 7];
